os/fs/smartfs/Kconfig

```diff
@@ -54,6 +54,9 @@ config SMARTFS_MULTI_ROOT_DIRS
 		during the "mksmartfs" command when the device is
 		initialized.
 
+		When this option is enabled AUTOMOUNT_USERFS_DEVNAME will
```

Comment:
Duplicated with line 50~53. No need to leave here.

Reply:
Removed that line in the below commit.